.github/workflows/delete-pr-build-on-close.yml

```diff
@@ -4,7 +4,7 @@ name: Delete pre-release when a branch is deleted
 # The circleci config builds CLI binaries when a PR is opened and hosts them under a GitHub (pre-)release named after the PR branch
 # This workflow action deletes that pre-release when a PR is merged or closed.
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
AFAICT this is the "same" event as `pull_request` but runs in the base branch context. That makes testing it within a PR not so simple I find 🤔

Reply:
Wow, good find!

suggestion: Always a good idea to leave a comment above around why we've using `pull_request_target:` otherwise a future maintainer may come in and change this back to `pull_request:`